watertap/property_models/activated_sludge/modified_asm2d_reactions.py

```diff
@@ -59,6 +59,28 @@ class ModifiedASM2dReactionParameterData(ReactionParameterBlock):
     Property Parameter Block Class
     """
 
+    CONFIG = ReactionParameterBlock.CONFIG()
+
+    CONFIG.declare(
+        "decay_switch",
+        ConfigValue(
+            default=DecaySwitch.on,
+            domain=In(DecaySwitch),
+            description="Switching function for decay",
+            doc="""
+           Switching function for handling decay in reaction rate expressions.
+
+           **default** - `DecaySwitch.on``
+
+       .. csv-table::
+           :header: "Configuration Options", "Description"
+
+           "``DecaySwitch.on``", "The decay of heterotrophs and autotrophs is dependent on the electron acceptor present"
+           "``DecaySwitch.off``", "The decay of heterotrophs and autotrophs does not change"
+       """,
+        ),
+    )
+
```

Comment:
I am a little confused about this configuration, it seems the default settings is `DecaySwith.on`, but I don't see formulas related to it, it seems `DecaySwitch.on` and `DecaySwitch.off` do not have differences in this code.

Reply:
Some of the reaction rates in this file differ depending on whether DecaySwitch is on or off.